packages/block-library/src/code/edit.js

```diff
@@ -6,17 +6,22 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { PlainText } from '@wordpress/block-editor';
+import {
+	PlainText,
+	__experimentalBlock as Block,
+} from '@wordpress/block-editor';
 
-export default function CodeEdit( { attributes, setAttributes, className } ) {
+export default function CodeEdit( { attributes, setAttributes } ) {
 	return (
-		<div className={ className }>
+		<Block.pre>
 			<PlainText
+				__experimentalVersion={ 2 }
+				tagName="code"
```

Comment:
The code change is good, I believe it maps the frontend which is a good thing but in my testing, the "code" block gets a background which I believe is the default background provided by WP Admin styles. I wonder what we can do about that.

Reply:
Yes, I'm aware of it. :/ We'll have to add this to the admin reset stylesheet.

Reply:
We will need #19837 to continue with this.

Reply:
Ok, this is merged.